tracer/test/Datadog.Trace.Security.Unit.Tests/WafIpBlockTests.cs

```diff
@@ -0,0 +1,184 @@
+// <copyright file="WafIpBlockTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.RcmModels.AsmData;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Security.Unit.Tests
+{
+    [Collection("WafTests")]
+    public class WafIpBlockTests
```

Comment:
Again, I think this is from the IP blocking PR

Reply:
As said above, I think it makes sense to test the waf returning a block actions independently of the blocking PR which will articulate RCM with it. Here we also test the Actions bindings are working, which is just part of waf 1.5